pkg/kubelet/config/file_linux.go

```diff
@@ -151,3 +155,30 @@ func (s *sourceFile) consumeWatchEvent(e *watchEvent) error {
 	}
 	return nil
 }
+
+func modifyStaticPodForPinnedManagement(pod *v1.Pod) {
+	if !managed.IsPodManaged(pod) {
+		return
+	}
+	if pod.Annotations == nil {
+		pod.Annotations = make(map[string]string)
+	}
+	if pod.Labels == nil {
+		pod.Labels = make(map[string]string)
+	}
+	pod.Labels[managed.WorkloadsLabel] = ""
+	for _, container := range pod.Spec.Containers {
+		if container.Resources.Requests.Cpu().IsZero() {
+			continue
+		}
+		var resourceLimit string
+		if val, valid := container.Resources.Requests.Cpu().AsInt64(); valid {
+			resourceLimit = fmt.Sprintf("%v", managed.MilliCPUToShares(val*1000))
```

Comment:
you can use `cpuRequest.MilliValue()` similar to https://github.com/kubernetes/kubernetes/blob/9c9af69ea60519b610a70261fb1b8bb81a2a5d04/pkg/kubelet/cm/helpers_linux.go#L126

Reply:
fixed. thanks